x-pack/test/case_api_integration/common/lib/authentication/index.ts

```diff
@@ -90,5 +90,3 @@ export const deleteSpacesAndUsers = async (getService: CommonFtrProviderContext[
   await deleteSpaces(getService);
   await deleteUsersAndRoles(getService);
 };
-
-export const superUserSpace1Auth = { user: superUser, space: 'space1' };
```

Comment:
Moved this to the utils file so it could leverage the `getAuthWithSuperUser` helper function.